testsuite/MDAnalysisTests/coordinates/test_netcdf.py

```diff
@@ -299,6 +300,23 @@ def test_box(self, universe, index, expected):
         assert_almost_equal(self.box_refs[expected], universe.dimensions)
 
 
+class TestNCDFReader4(object):
+    """NCDF Trajectory exported by cpptaj, without `time` variable.
+    """
+    prec = 3
+
+    @pytest.fixture(scope='class')
+    def u(self):
+        return mda.Universe(CPPTRAJ_TRAJ_TOP,
```

Comment:
What happens if you pass `continuous=True` here? Does it fix the sequential numbering issue?

Reply:
There is no more sequential number in issue as I followed your suggestion to not set the time at all. The default behavior correctly handled the sequential trajectories!

Reply:
Awesome, great to hear the chain reader is not completely broken 😅 !